### DISCUSSION TITLE
2024 Open Visualization Collaborator Summit (September 10-11, London)

### DISCUSSION THREAD

**Discussion by chrisgervang** [visgl/deck.gl#8835] [Announcement]
We’re excited to announce that this year’s Collab Summit will be held at the [Microsoft Reactor Space](https://developer.microsoft.com/en-us/reactor/about/) in central London on September 10th and 11th - near the iconic Paddington station in the City of Westminster!

For a detailed schedule, registration details, and more information, please see the [main post about the Collaborator Summit](https://github.com/openjs-foundation/summit/issues/407).

## Registration

This is an open event, but is geared towards contributors of the deck.gl framework and other frameworks in the vis.gl and kepler.gl OpenJS projects.

Registration is required for attendance at the Summit. To attend the in-person Summit in London, please register here:

https://linuxfoundation.regfox.com/open-visualization-collaborator-summit-2024

##  Speaking

We invite contributors within the vis.gl / deck.gl / kepler.gl visualization ecosystem to submit sessions [through this link](https://github.com/openjs-foundation/summit/issues/new?assignees=chrisgervang&labels=New+York+2023%2C+Collaborator+Summit%2C+Session+Proposal&projects=&template=openviz-summit-topic-proposal.md&title=Open+Visualization+Session+Proposal%3A+%3CProject+Name%3E+%3CSession+Title%3E). We encourage submissions that foster collaboration, including panels, fire-side chats, discussions, interactive workshops, and presentations.

Have an idea for a talk and want to run it by us? Discuss it here or contact us at [openvizsummit@openjs.org](mailto:openvizsummit@openjs.org) if you’re interested.

## Volunteering 

Interested in helping organize this event? We’re assembling a small team of volunteers and have outlined various roles you can contribute to in these [slides](https://docs.google.com/presentation/d/1b5R1UQEPW-jC1RpmQtuNS2wWeg7dkOAO_pIBDv-6B1U/edit). Please contact @chrisgervang if you’re interested.

**In particular, we're looking for someone to create a London-themed visualization to feature on our website, similar to what we did for the [Madrid Summit in 2022](https://deck.gl/events/madrid-summit-2022/).**

![image](https://github.com/visgl/deck.gl/assets/2461547/bd4ee3cf-3274-4b77-9ff0-cad779f413b0)

We look forward to seeing everyone in September! :gb:

**Comment by Pessimistress** [comment]
### [Event Agenda](https://deck.gl/events/london-summit-2024/index.html)

Reminder to register for the event!
